azerothcore/azerothcore-wotlk
# feat(Core): Another way to get talent points

Aim to add support of mod-army-rank module
https://github.com/zxdhe121/mod-army-rank.git

### HOW TO TEST THE CHANGES:

<s>
Apply this PR
Added module mod-army-rank https://github.com/zxdhe121/mod-army-rank.git
Summon the NPC ID=61089, and buy items from him.
Use the items.
Check your talent page
Relogin and check again
</s>

**Edit by Shin**: as the above has already been tested by the PR author, to merge this PR we need to make sure that this PR does not alter the current talent system in any way. So:

- Check the talent system and make sure it still works fine
- Try to levelup and check that the correct amount of talent points is assigned
- Also check the DK class has its talent progression is different
